### PR TITLE
Footer-Small with childfragments

### PR DESCRIPTION
**Pull request by raphaellueckl**
The footer-small, now with childelements instead of modals.

### CODE REVIEW

**Review by 01231 — COMMENTED** [review]
In my opinion it would make sense to have a default active - for example the first one. What do you think about this?

File: src/components/20-molecules/footer-small/demo.js [hunk 1]
```diff
@@ -2,57 +2,87 @@
...
+
+    const dataProtectionLink = document.createElement('a');
+    dataProtectionLink.slot = 'disclaimer-item';
+    dataProtectionLink.href =
+      'https://axa.ch/en/information/data-protection.html';
+    dataProtectionLink.textContent = 'Data protection';
+
+    const copyRightText = document.createElement('span');
+    copyRightText.slot = 'copyright';
+    copyRightText.textContent = '© 2019 AXA Insurance Ltd.';
```
Comment (01231) [inline src/components/20-molecules/footer-small/demo.js]: use HTML entity `copyRightText.innerHTML = '&copy; 2019 AXA Insurance Ltd.';`

File: src/components/20-molecules/footer-small/story.js [hunk 2]
```diff
@@ -11,39 +11,36 @@ storiesOf('Molecules/Footer Small', module)
...
+      >
+        Terms of use
+      </a>
+      <a
+        slot="disclaimer-item"
+        href="https://axa.ch/en/information/data-protection.html"
+      >
+        Data protection
+      </a>
+      <span slot="copyright">© 2019 AXA Insurance Ltd.</span>
```
Comment (01231) [inline src/components/20-molecules/footer-small/story.js]: use HTML entity `&copy; 2019 AXA Insurance Ltd.`

File: src/components/20-molecules/footer-small/react/FooterSmall.jsx [hunk 3]
```diff
@@ -4,44 +4,57 @@ import createAXAFooterSmallReact from '../index.react';
...
+        >
+          Terms of use
+        </a>
+        <a
+          slot="disclaimer-item"
+          href="https://axa.ch/en/information/data-protection.html"
+        >
+          Data protection
+        </a>
+        <span slot="copyright">© 2019 AXA Insurance Ltd.</span>
```
Comment (01231) [inline src/components/20-molecules/footer-small/react/FooterSmall.jsx]: use HTML entity `&copy; 2019 AXA Insurance Ltd.`

File: src/components/20-molecules/footer-small/index.js [hunk 4]
```diff
@@ -20,8 +22,6 @@ class AXAFooterSmall extends LitElement {
     return {
       languageItems: { type: Array },
       disclaimerItems: { type: Array },
```
Comment (01231) [inline src/components/20-molecules/footer-small/index.js]: you can delete this two badboys

File: src/components/20-molecules/footer-small/index.js [hunk 5]
```diff
@@ -30,72 +30,88 @@ class AXAFooterSmall extends LitElement {
     super();
     this.languageItems = [];
     this.disclaimerItems = [];
```
Comment (01231) [inline src/components/20-molecules/footer-small/index.js]: same here

File: src/components/20-molecules/footer-small/ui.test.js [hunk 6]
```diff
@@ -71,30 +73,36 @@ test('should have a link in the href attribute', async t => {
...
-      .shadowRoot.querySelector('div[class*="js-footer-small__copyright"]')
+  const copyright = Selector(
+    () =>
+      document
+        .querySelector(`axa-footer-small`)
+        .shadowRoot.querySelector('slot[name="copyright"]')
+        .assignedNodes()[0]
   );
   await t.expect(copyright.exists).ok();
   await t.expect(copyright.textContent).eql('© 2019 AXA Insurance Ltd.');
```
Comment (01231) [inline src/components/20-molecules/footer-small/ui.test.js:108]: this is the same test as above

**Review by 01231 — CHANGES_REQUESTED** [review]

File: src/components/20-molecules/footer-small/ui.test.js (hunk 6 above)
Comment (raphaellueckl) [reply src/components/20-molecules/footer-small/ui.test.js:108]: This is just a smoke test to basically see that it works in react.

Comment (01231) [reply src/components/20-molecules/footer-small/ui.test.js:108]: ah oke did not see it is for react

**Review by 01231 — APPROVED** [review]
👍

**Pull request merged**